src/main/scala/uk/gov/hmrc/ct/accounts/frs10x/abridged/AC5052A.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ct.accounts.frs10x.abridged
+
+import uk.gov.hmrc.ct.accounts.AccountsMoneyValidation
+import uk.gov.hmrc.ct.accounts.frs10x.retriever.Frs10xAccountsBoxRetriever
+import uk.gov.hmrc.ct.box._
+
+
+case class AC5052A(value: Option[Int]) extends CtBoxIdentifier(name = "Debtors due after more than one year") with CtOptionalInteger
+                                                                                                              with Input
+                                                                                                              with ValidatableBox[Frs10xAccountsBoxRetriever] {
+
+  override def validate(boxRetriever: Frs10xAccountsBoxRetriever): Set[CtValidation] = validateIntegerRange("AC5052A", this, 0, 99999999)
```

Comment:
James had added new trait called `AccountsMoneyValidation` which has these max numbers built into it. I've changed to take in a min, I'll send the code to you as I'm still working on it.